network/model.go

```diff
@@ -12,7 +12,28 @@ type CreateGameRequest struct {
 	PlayerIDs []model.PlayerID `json:"playerIDs"`
 }
 
+// TODO figure out the minimum info the client will need
+type CreateGameResponse struct {
+	Players         []model.Player                       `json:"players"`
+	PlayerColors    map[model.PlayerID]model.PlayerColor `json:"player_colors,omitempty"`
```

Comment:
for json tags: is it easier to use snake_case or camelCase? what's the argument either way?

Reply:
Neither is easier; I was under the assumption that snake_case is more conventional for JS :shrug: